### PR TITLE
Improve the readability of the code and update the default parameters

### PR DESCRIPTION
**Pull request by krishanthisera**
### Context
This pull request aims to address some minor concerns with the pipe.

### Changes
- Added a new warning handler and handled the warnings
- Let the dependency checker extendible by passing the `EXTRA_ARGS` deployment  variable
- Include `--prettyPrint` as a parameter to the dependency checker
- Some minor changes to improve the code readability

### CODE REVIEW

File: pipe/pipe.py [hunk 1]
```diff
@@ -3,47 +3,61 @@
...
+    'OUTPUT_PATH': {'type': 'string', 'required': False, 'default': './test-results/'},
+    'OSSINDEX_USERNAME': {'type': 'string', 'required': False, 'default': ''},
+    'OSSINDEX_PASSWORD': {'type': 'string', 'required': False, 'default': ''},
+    'DISABLE_OSSINDEX': {'type': 'boolean', 'required': False, 'default': False},
+    'EXTRA_ARGS': {'type': 'string', 'required': False, 'default': ''},
     }
 
 class OWASPDependencyCheck(Pipe):
-    owasp_failure = False;
+    owasp_failure   = False;
```
Comment (TheOrangePuff) [inline pipe/pipe.py:25]: I'm not super familiar with python dev tools but is there some sort of linter or prettier equivalent to pick things like this up?

File: pipe/pipe.py [hunk 2]
```diff
@@ -63,6 +77,10 @@ def run_owasp_check(self):
         if self.disableOssIndex == True:
             owasp_command.append('--disableOssIndex')
 
+        if self.extra_args:
+            warnings.warn(f"Dependency check has been extended: {self.extra_args}", UserWarning)
```
Comment (TheOrangePuff) [inline pipe/pipe.py:81]: Do we need a warning here? :thinking:

**Review by TheOrangePuff — CHANGES_REQUESTED** [review]
Just a couple questions but looks good!

File: pipe/pipe.py (hunk 1 above)
Comment (krishanthisera) [reply pipe/pipe.py:25]: I haven't used such a tool. But wanted to keep the `=` aligned.
image

File: pipe/pipe.py (hunk 2 above)
Comment (krishanthisera) [reply pipe/pipe.py:81]: I think we do need some indication if we should extend the dependency checker.

File: pipe/pipe.py (hunk 1 above)
Comment (TheOrangePuff) [reply pipe/pipe.py:25]: Ah I see!

File: pipe/pipe.py (hunk 2 above)
Comment (TheOrangePuff) [reply pipe/pipe.py:81]: I'm not sure what you mean by extending the dependency checker?

**Review by TheOrangePuff — APPROVED** [review]
LGTM!

**Pull request merged**